app/views/asset/business/BusinessDescriptionView.scala.html

```diff
@@ -15,33 +15,36 @@
  *@
 
 @import controllers.asset.business.routes._
+@import views.html.components.{ErrorSummary, InputText, SubmitButton}
 
 @this(
     main_template: MainTemplate,
-    formHelper: FormWithCSRF
+    formHelper: FormWithCSRF,
+    errorSummary: ErrorSummary,
+    inputText: InputText,
+    submitButton: SubmitButton
 )
 
 @(form: Form[_], draftId: String, index: Int, businessName: String)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = s"${errorPrefix(form)} ${messages("business.description.title")}"
+    title = s"${errorPrefix(form)} ${messages("business.description.title")}",
+    showBackLink = true
 ) {
 
     @formHelper(action = BusinessDescriptionController.onSubmit(index, draftId), 'autoComplete -> "off") {
 
-        @components.back_link()
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
-
-        @components.input_text(
+        @inputText(
           field = form("value"),
           label = messages("business.description.heading", businessName),
           labelAsHeading = true,
-          labelClass=Some("heading-large"),
+          labelClass=Some("govuk-heading-l"),
           inputClass = Some("form-control-3-4"),
```

Comment:
inputClass looks like an old one to control the size, I think the equivalent would be `govuk-!-width-three-quarters`

Reply:
Done